lang/java/pom.xml

```diff
@@ -472,7 +472,6 @@
         <groupId>joda-time</groupId>
         <artifactId>joda-time</artifactId>
         <version>${joda.version}</version>
-        <optional>true</optional>
```

Comment:
This is fine. It is optional in lang/java/avro/pom.xml and shouldn't be optional for the specific compiler.